components/dictionary.txt

```diff
@@ -25,26 +28,32 @@ chatbot
 cheatsheet
 cheatsheets
 checkmarks
+CLI
 compositional
 conda
 COVID
 customizable
 Cynwyd
+dir
 discoverable
 DMPTool
 Dockerfiles
 DoubleTree
 Drexel
 dropdown
+ectory
 EmptyDrops
 endcase
 endcomment
 endif
 endfor
 Ensembl
 epitope
+errored
 et
 familiaris
+fastq
+FASTQ
```

Comment:
I would check how fastq/FASTQ are being used, as I would generally expect us to want to standardize on one.